webhooks-extension/endpoints/webhook.go

```diff
@@ -42,6 +42,16 @@ func (r Resource) createWebhook(request *restful.Request, response *restful.Resp
 		RespondError(response, err, http.StatusBadRequest)
 		return
 	}
+
+	if webhook.ReleaseName != "" {
+		if len(webhook.ReleaseName) > 63 {
+			err := errors.New("Wanted release name is too long, must be less than 64 characters")
+			log.Printf("Error: %s.", err.Error())
```

Comment:
Wondering if we need an issue so we have different logging levels .... errors would normally start lower cased.